assets/scss/components/stencil/navPages/_navPages.scss

```diff
@@ -320,7 +321,7 @@
         @include breakpoint("medium") {
             border-left: 1px solid stencilColor("navPages-subMenu-separatorColor");
             display: inherit;
-            height: $navPage-subMenu-horizontal-height;
```

Comment:
Is this variable being used anywhere else ? Or can we remove the definition as well completely ?